backend/services/prisonerProfileService.js

```diff
@@ -142,9 +144,11 @@ module.exports = ({
 
     const canViewSocLink = Boolean(isSocUser && socDetails)
 
-    const complexityLevel = complexity.enabled && (await complexityApi.getComplexOffenders(context, [offenderNo]))
+    const complexityLevel =
+      isComplexityEnabledFor(agencyId) && (await complexityApi.getComplexOffenders(context, [offenderNo]))
+
     const isHighComplexity =
-      complexity.enabled && Boolean(complexityLevel?.length > 0 && complexityLevel[0]?.level === 'high')
+      isComplexityEnabledFor(agencyId) && Boolean(complexityLevel?.length > 0 && complexityLevel[0]?.level === 'high')
```

Comment:
i don't think you need the call to `isComplexityEnabledFor` here - the `complexityLevel` will not be set if complexity is not enabled for this agency

Reply:
Ah, good point.